cdk/src/open-data-platform/data-plane/data-import/write-violations-data.handler.ts

```diff
@@ -46,13 +46,18 @@ const insertIntoStatement = `INSERT INTO epa_violations (violation_id,
                                                          violation_code,
                                                          compliance_status,
                                                          start_date,
-                                                         end_date)
-                             VALUES (:violation_id,
-                                     :pws_id,
-                                     :violation_code,
-                                     :compliance_status,
-                                     :start_date,
-                                     :end_date) ON CONFLICT (violation_id) DO NOTHING`;
+                                                         end_date,
+                                                         state_census_geo_id)
+                             SELECT :violation_id,
+                                    :pws_id,
+                                    :violation_code,
+                                    :compliance_status,
+                                    TO_DATE(:start_date, 'YYYY-MM-DD'),
+                                    TO_DATE(:end_date, 'YYYY-MM-DD'),
```

Comment:
What happens here if end_date is null?

Reply:
never null now since I changed to default to empty string when null.